scripts/jest/jest-cli.js

```diff
@@ -0,0 +1,130 @@
+'use strict';
+
+const {spawn} = require('child_process');
+const chalk = require('chalk');
+const yargs = require('yargs');
+
+const configMap = {
+  source: './scripts/jest/config.source.js',
+  www: './scripts/jest/config.source-www.js',
+  persist: './scripts/jest/config.source-persistent.js',
+  build: './scripts/jest/config.build.js',
+  devtools: './scripts/jest/config.build-devtools.js',
+};
+
+const argv = yargs
+  .parserConfiguration({
+    // Important: This option tells yargs to move all other options not
+    // specified here into the `_` key. We use this to send all of the
+    // Jest options that we don't use through to Jest (like --watch).
+    'unknown-options-as-args': true,
+  })
+  .options({
+    debug: {
+      alias: 'd',
+      describe: 'Run with node debugger attached.',
+      requiresArg: false,
+      type: 'boolean',
+      default: false,
+    },
+    prod: {
```

Comment:
There are three different build types: `prod`, `dev`, and `profiling`. We don't currently run tests against the profiling builds, but we might in the future. So we should implement this as an enum.

You can keep `--prod` as an alias, though.

Reply:
IMO we can iterate on this. We weren't already running anything against profiling builds, so the new script doesn't really need to support this out of the box.

That being said, I would assume `buildType=prod|dev|profiling` with maybe `--prod`, `--dev`, and `--profiling` shortcut flags as optional syntactic sugar if you really felt like it.